src/components/HelloWorld.vue

```diff
@@ -6,33 +6,31 @@
           :src="require('../assets/logo.svg')"
           class="my-3"
           contain
-          height="200"
```

Comment:
any reason for height change?

Reply:
Height of the vuetify image was too large, made it smaller